app/data/models/settlement.js

```diff
@@ -4,12 +4,14 @@ module.exports = (sequelize, DataTypes) => {
     paymentRequestId: DataTypes.INTEGER,
     detail: DataTypes.STRING,
     invalid: DataTypes.BOOLEAN,
```

Comment:
invalid should be after generated. Sorry, this not something you have just done.